### PR TITLE
Create subscription from registration

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: registration/tasks.py [hunk 1]
```diff
@@ -127,7 +235,7 @@ def jembi_post_json(registration_id):
         try:
             result = requests.post(
                 "%s/json/subscription" % settings.JEMBI_BASE_URL,  # url
-                headers={'Content-Type': 'application/json'},
+                headers={'Content-Type': ['application/json']},
```
Comment (imsickofmaps) [inline registration/tasks.py]: this was correct before, why did you change it?

**Comment by imsickofmaps** [comment]
Looking good, just missing the tests to see did the calcs make it into the subscription models correctly

File: registration/tests.py [hunk 2]
```diff
@@ -15,14 +16,14 @@
 from registration import tasks
 
 
-def override_get_timestamp():
-    return "20130819144811"
+def override_get_today():
+    return datetime.strptime("20130819144811", "%Y%m%d%H%M%S")
```
Comment (imsickofmaps) [inline registration/tests.py:21]: implement test date alignment now please

Comment (gsvr) [inline registration/tests.py:21]: Implemented - changed 'today' date rather than timestamp for simplicity.

**Comment by gsvr** [comment]
@imsickofmaps Ready for re-review - have a look at the last 3 commits.

**Comment by imsickofmaps** [comment]
minor simplify then :+1: when travis happy
